src/Serialization/main.jl

```diff
@@ -365,6 +368,51 @@ macro register_serialization_type(ex::Any, args...)
   return register_serialization_type(ex, str, uses_id, uses_params)
 end
 
+
+################################################################################
+# Utility macro
+"""
+    @import_all_serialization_functions
+
+This macro imports all serialization related functions that one may need for implementing
+serialization for custom types from Oscar into the current module.
+One can instead import the functions individually if needed but this macro is provided
+for convenience.
+"""
+macro import_all_serialization_functions()
```

Comment:
This is the macro that we talked about this afternoon. If you have a better name suggestion, please use that instead.
And I think it would be good if you look through the list below to see if these are indeed the functions that users should/may use to implement their own serialization (inspired by the already existing serializations in Oscar)